### DISCUSSION TITLE
Add an option to toggle country/region flag display

### DISCUSSION THREAD

**Discussion by Stehsaer** [ppy/osu#25956] [Ideas] [answered]
Certain country/region flags are illegal in certain countries/regions and are not allowed to display publicly. To help players obey the law in their countries/regions and to help them obey the rule of the platform they are streaming on, I'd suggest adding a switch or an option to choose whether to display flags on leaderboards, friend lists, or any other place that contains a country/region flag.

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/discussions/24369
